lib/paramedic.js

```diff
@@ -149,21 +149,22 @@ ParamedicRunner.prototype = {
 
 var storedCWD =  null;
 
-function parseTargetsFromConfig(paramedicConfigPath) {
-    var config = JSON.parse(fs.readFileSync(paramedicConfigPath, 'utf8'));
-    return config.targets.map(function(target) {
+function parseConfig(paramedicConfigPath) {
```

Comment:
Please wrap configuration parsing/etc logic to a separate file (hide implementation details): 
```
var config = ParamedicConfig (path[optional])
config.getTestTargets();
config.getTestPlugins()'
...
```